app/views/work_days/_form.html.erb

```diff
@@ -26,19 +29,19 @@
   <div class="row">
     <div class="col-md-4">
       <div class="form-group">
-        <%= f.label :start_time, class: "control-label"  %><br>
+        <%= f.label :start_time, class: "control-label" %><font color="red">*</font><br>
```

Comment:
Passen hier (und in den folgenden Zeilen) einfache Anführungszeichen nicht besser?